packages/shopify-codemod/transforms/global-assignment-to-default-export.js

```diff
@@ -29,10 +47,24 @@ export default function globalAssignmentToDefaultExport({source}, {jscodeshift:
 
           if (expose == null) {
             expose = j(member).toSource();
+            return j.exportDefaultDeclaration(statement);
           } else {
-            throw new Error('Found multiple exports in a single file, please break up the file first');
+            const newExpose = j(member).toSource();
+
+            if (!isDefaultExportMemberAssignment(expose, newExpose)) {
+              throw new Error('Found multiple exports in a single file, please break up the file first');
+            }
+
+            const {left, right} = path.node.expression;
+
+            return j.expressionStatement(
+              j.assignmentExpression(
+                '=',
```

Comment:
I'd probably use the original operator, just to move all of the decision making on what kind of assignment in one spot (the find call).